api/v1beta1/activemqartemis_types.go

```diff
@@ -377,6 +378,52 @@ type DeploymentPlanType struct {
 	// Specifies the container security context
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Container Security Context"
 	ContainerSecurityContext *corev1.SecurityContext `json:"containerSecurityContext,omitempty"`
+	// Specifies additional Volumes to be attached to the broker pods
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Volumes"
+	ExtraVolumes []corev1.Volume `json:"extraVolumes,omitempty"`
+	// Specifies mount options for extraVolumes
+	// +optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Mount Options"
+	ExtraVolumeMounts []corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
+	// Specifies Extra Volume Claims Templates for the broker pods
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Volume Claims Templates"
+	ExtraVolumeClaimTemplates []VolumeClaimTemplate `json:"extraVolumeClaimTemplates,omitempty"`
+}
+type VolumeClaimTemplate struct {
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Metadata"
+	ObjectMeta `json:"metadata,omitempty"`
+
+	// spec defines the desired characteristics of a volume requested by a pod author.
+	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#persistentvolumeclaims
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Spec"
+	Spec corev1.PersistentVolumeClaimSpec `json:"spec,omitempty"`
+}
+
+type ObjectMeta struct {
+	// Name must be unique within a namespace. Is required when creating resources, although
+	// some resources may allow a client to request the generation of an appropriate name
+	// automatically. Name is primarily intended for creation idempotence and configuration
+	// definition.
+	// Cannot be updated.
+	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#names
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Name"
+	Name string `json:"name,omitempty"`
```

Comment:
The field name doesn't define any xDescriptors
```suggestion
	// Name must be unique within a namespace. Is required when creating resources, although
	// some resources may allow a client to request the generation of an appropriate name
	// automatically. Name is primarily intended for creation idempotence and configuration
	// definition.
	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#names
	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Name",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
	Name string `json:"name,omitempty"`
```

Reply:
+1